From: Helmut Grohne <helmut@subdivi.de>
Date: Mon, 3 Apr 2023 18:13:03 +0200
Subject: cross

---
 src/makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/makefile b/src/makefile
index d8ae170..b2edeb6 100644
--- a/src/makefile
+++ b/src/makefile
@@ -89,7 +89,7 @@ CC=gcc
 # c++ compiler
 CXX=g++
 # linker
-LD=g++
+LD=$(CXX)
 
 
 ############################# COMPILER FLAGS ##################################
