Subject: Fix undefined behavior (out of bounds write)
Origin: chuck-1.3.5.2-43-g6adcffa
Upstream-Author: Ivan Tarasov <Ivan.Tarasov@gmail.com>
Date: Sun Mar 13 03:03:20 2016 -0700

--- a/src/ugen_xxx.cpp
+++ b/src/ugen_xxx.cpp
@@ -2021,7 +2021,7 @@
         t_CKINT i;
         
         for ( i = 0 ; i < bufsize ; i++ ) buffer[i] = 0;
-        for ( i = 0 ; i < 3 ; i++ ) { acoeff[i] = 0; bcoeff[i] = 0; }
+        for ( i = 0 ; i < 2 ; i++ ) { acoeff[i] = 0; bcoeff[i] = 0; }
         
         acoeff[0] = 1.0;
         acoeff[1] = -.99;
